README.md

```diff
@@ -116,58 +112,80 @@ If you need to run additional shell commands before or after `wrangler publish`,
 jobs:
   deploy:
     steps:
-      uses: cloudflare/wrangler-action@1.3.0
+      uses: cloudflare/wrangler-action@2.0.0
       with:
         apiToken: ${{ secrets.CF_API_TOKEN }}
+        command: publish
         preCommands: echo "*** pre command ***"
         postCommands: |
           echo "*** post commands ***"
           wrangler kv:key put --binding=MY_KV key2 value2
           echo "******"
 ```
 
-Set the optional `publish` input to false to skip publishing your Worker project and secrets.  Useful in conjunction with pre and post commands.  For example, if you only wanted to run `wrangler build` against your project:
+You can use the `command` option to do specific actions such as running `wrangler build` against your project:
```

Comment:
`wrangler build` is no longer a thing in Wrangler 2.
Perhaps remove this sentence?

Reply:
I think it's still good to have this little bit, should just be a different command. Not sure what that command should be right now though...